generators/guestcontainer/templates/Service/ServiceManifestWithPorts.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ServiceManifest Name="<%= servicePkgName %>" Version="1.0.0"
+                 xmlns="http://schemas.microsoft.com/2011/01/fabric" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" >
+
+   <ServiceTypes>
+      <StatelessServiceType ServiceTypeName="<%= serviceTypeName %>" UseImplicitHost="true">
+   </StatelessServiceType>
+   </ServiceTypes>
+   
+   <CodePackage Name="code" Version="1.0.0">
+      <EntryPoint>
+         <ContainerHost>
+            <ImageName><%= imageName %></ImageName>
+            <Commands><%= commands %></Commands>
+         </ContainerHost>
+      </EntryPoint>
+      <EnvironmentVariables> 
+      </EnvironmentVariables> 
+   </CodePackage>
+   <Resources>
+        <Endpoints>
+            <Endpoint Name="<%= serviceEndPointName %>" UriScheme="http" Port="<%= portMapHost %>" Protocol="http"/>
```

Comment:
We are good for now, but just thinking, cant we do away without duplicating the manifests? Imagine, tomorrow when you have to support volumes, you will have lot of permuations. May be we add a placeholder where we replace the entire port mapping line and endpoint line. Something like that.

Reply:
Yes i tried doing it that way. But i was facing some issue because the copytp1 (method which copies appmanifest from template to desired location) after the complete code executes and the placeholder code is getting executed before the template is actually getting generated. 
So when with this approach. Will do a do a deeper investigation on how to do this in future.